e2e/stateful-pallet-storage/handleItemized.test.ts

```diff
@@ -29,31 +30,47 @@ describe('📗 Stateful Pallet Storage Itemized', function () {
   let badMsaId: u64;
 
   before(async function () {
-    // Create a provider for the MSA, the provider will be used to grant delegation
-    [providerKeys, providerId] = await createProviderKeysAndId(fundingSource, 2n * DOLLARS);
+    [
+      // Create a provider for the MSA, the provider will be used to grant delegation
+      [providerKeys, providerId],
+      // Delegator Keys
+      delegatorKeys,
+      schemaId_deletable,
+      schemaId_unsupported,
+    ] = await Promise.all([
+      createProviderKeysAndId(fundingSource, 2n * DOLLARS),
```

Comment:
I wouldn't change it now but how many of these could/should this be batched too?